src/app/pages/credentials/credentials.component.html

```diff
@@ -5,7 +5,6 @@ <h3>Misc</h3>
         <mat-divider></mat-divider>
         <ul>
             <li (click)="systemNav('email')">Email</li>
-            <li (click)="navExternal('https://www.truenas.com/docs/')">Guide</li>
```

Comment:
if there is no usage of `navExternal` method do we need to keep it?

Reply:
Good idea. Removed.